features/lwipstack/LWIPInterface.cpp

```diff
@@ -267,9 +272,21 @@ char *LWIP::Interface::get_mac_address(char *buf, nsapi_size_t buflen)
     return buf;
 }
 
-char *LWIP::Interface::get_ip_address(char *buf, nsapi_size_t buflen)
+char *LWIP::Interface::get_interface_name(char *buf)
+{
+    sprintf(buf, "%c%c%d", netif.name[0], netif.name[1], netif.num);
```

Comment:
Make sure that netif structure has enough space for this (6 chars)

Reply:
netif.num is u8_t